### PR TITLE
Lab 3

### PR DESCRIPTION
**Pull request by Carl-Hugo**
Lab 3 - Work in progress:

- Add the first steps

### CODE REVIEW

**Comment by fboucher** [comment]
@Carl-Hugo  since lab, I like it.
I would change your sub-titles to match your initial "agenda":

`In this lab, we will execute the following steps:

Make our project ready for docker
Build our first Docker image
Run our dockerized application locally
Learn how to stop our running container
Deploy our dockerized application to Azure`

I would also like to see/know what the manual docker command behind the steps we are doing with VSCode.  The tool is nice and help us to be efficient, but not required.  

So maybe do an intro at the beginning saying that every step you will also provide all the commands and then just put a code snippet... or something like that.

Otherwise great job!

**Comment by Carl-Hugo** [comment]
@FBoucher good point, I will update the "steps name" and add the Docker commands as well.

**Comment by Carl-Hugo** [comment]
Done the corrections and added the manual steps; I only need to add the `Deploy our dockerized application to Azure` section, and this will be done (probably more next weekend tho).

**Comment by Carl-Hugo** [comment]
@FBoucher Finally, I was feeling like it this afternoon, so I completed Lab3. 

If you want to try it out, you can:

1. Follow it step by step
1. Follow the first few steps then copy `Lab3.ps1` to a project containing a `Dockerfile`; then from there run : 
    ```powershell
    .\lab3.ps1 -containerRegistryName <containerRegistryName> -containerDnsName <containerDnsName>
    ```
    The value I used where: 
    ```powershell
    .\lab3.ps1 -containerRegistryName GAB2019ContainerRegistry -containerDnsName gab-2019-container-demo
    ```
    > You may need to `az login` first.

If the lab is OK for you guys, I'll resolve `Lab0/README.md` conflicts (reverting my changes basically) and complete the PR.

File: Lab3/README.md [hunk 1]
```diff
@@ -4,17 +4,339 @@
...
+
+#### Using the Docker CLI
+
+To create a Docker image manually:
+
+1. Open a terminal
+    - In VS Code you can `` ctrl+` `` or click `View > Terminal`.
+        > You can even have multiple terminals opened at the same time. On Windows 10, they can be `cmd`, `PowerShell` and even `bash` if you have a Linux subsystem installed!
+    - In Windows 10 you can open PowerShell by `right-click Start Menu > Windows PowerShell` or from Explorer (`Windows+E`): `File > Open Windows PowerShell`.
+    - In MacOS or Linux open `bash`. TODO: how to do that?
```
Comment (Carl-Hugo) [inline Lab3/README.md:134]: How to open bash in MacOS/Linux?

**Pull request merged**